templates/backtomap.mustache

```diff
@@ -0,0 +1,39 @@
+{{!
+    This file is part of Moodle - https://moodle.org/
+
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template mod_learningmap/backtomap
+
+    Template for displaying the "back to learning map" button in activity descriptions.
+
+    Classes required for JS:
+    * none
+
+    Data attributes required for JS:
+    * none
+
+    Context variables required for this template:
+    * none
+
+    Example context (json):
+    {
+    "url": "https://moodle.localhost/mod/learningmap/view.php?id=1234",
+    "name": "Learningmap"
+    }
+}}
+<div class="learningmap-backlink">
+    <a class="btn btn-primary" href="{{{url}}}">{{#str}} backlink, learningmap, { "name": {{#quote}}{{name}}{{/quote}} } {{/str}}</a>
```

Comment:
The Back button is not so important, unlike Save. I would suggest to insert it as a secondary button,